libafl/src/executors/hooks/unix.rs

```diff
@@ -204,17 +204,18 @@ pub mod unix_signal_handler {
                 let mut bsod = Vec::new();
                 {
                     let mut writer = std::io::BufWriter::new(&mut bsod);
-                    writeln!(writer, "input: {:?}", input.generate_name(0)).unwrap();
-                    libafl_bolts::minibsod::generate_minibsod(
+                    let _ = writeln!(writer, "input: {:?}", input.generate_name(0));
+                    let _ = libafl_bolts::minibsod::generate_minibsod(
```

Comment:
Might be good to log this if it's `Err`? Else we may have bugs in minibsod that we never find because we eat them up